### DISCUSSION TITLE
audio visualizer widget

### DISCUSSION THREAD

**Discussion by opu-hossain** [prasanthrangan/hyprdots#1632] [Q&A] [answered]
I'm using the latest HyDE dotfiles for my Arch Linux configuration, 

I want to add a cool audio visualizer widget to my screen. but I don't know how to do it. Would anyone be able to help me with this?

I'm currently using a flat hub package
https://flathub.org/apps/org.nickvision.cavalier

but this is just an application. I want a widget

**Comment by AKA-Akhil** [comment, marked as answer]
why not use cava?
if you want a widget you can make one using eww
and if you want on one the waybar then you can just add it in the config file, you can just follow this [waybar wiki for cava](https://github.com/Alexays/Waybar/wiki/Module:-Cava)

**Reply by opu-hossain** [reply]
that helps, thank you!

**Reply by opu-hossain** [reply]
sure
